frontend/download-openapi-jar.sh

```
@@ -57,31 +29,31 @@ echo " => Selected version in config-file: $VER"
+
+echo " => Downloading .jar file using Maven:"
 echo "    Target-file: $DIR/$LOCAL_JAR"
 
-if command -v curl >/dev/null 2>&1; then
-  echo "  => Using curl"
-  curl -fSL "$URL" -o "$DIR/$LOCAL_JAR"
-else
-  echo "  => Using wget"
-  wget -O "$DIR/$LOCAL_JAR" "$URL"
-fi
+mvn dependency:copy -Dartifact=org.openapitools:openapi-generator-cli:$VER:jar -DoutputDirectory=$DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)